lib/serialisers/json-0.6.js

```diff
@@ -143,8 +143,13 @@ module.exports = createClass({
       element.attributes.remove('samples');
 
       if (samples) {
-        samples = samples.map(function(sample) {
-          return sample.get(0);
+        // Flatten samples array of array
+        var existingSamples = samples;
+        samples = new ArrayElement();
+        existingSamples.forEach(function(sample) {
+          sample.forEach(function (sample) {
+            samples.push(sample);
+          });
```

Comment:
That won't work because we have an Array Element of Array Element. The lodash flatten only supports `Array` unfortunately.